cea/technologies/thermal_network/thermal_network_matrix.py

```diff
@@ -1600,37 +1603,29 @@ def read_properties_from_buildings(building_names, buildings_demands, property):
 # ============================
 
 
-def run_as_script(scenario_path=None):
+def main(config):
     """
     run the whole network summary routine
     """
-    import cea.globalvar
-    import cea.inputlocator as inputlocator
     gv = cea.globalvar.GlobalVariables()
-
-    import cea.config
     config = cea.config.Configuration()
```

Comment:
this is a bug. you already have a variable called `config`. you shouldn't be creating your own instances of `cea.config.Configuration()` anyway...